projects/1.16/assets/evilcraft/evilcraft/lang/zh_cn.json

```
@@ -1 +1,765 @@
+  "advancement.evilcraft.master_distorter.desc": "一次用力场扭曲锤杀死超过10只生物。",
+  "advancement.evilcraft.spirit_cooker": "血腥厨师",
+  "advancement.evilcraft.spirit_cooker.desc": "搭建一个灵魂炉。",
+  "advancement.evilcraft.closure": "永恒封印",
+  "advancement.evilcraft.closure.desc": "使用复仇之焦捕捉一个复仇灵魂。",
+  "advancement.evilcraft.player_distorter": "扭曲玩家",
+  "advancement.evilcraft.player_distorter.desc": "用力场扭曲锤杀死玩家。",
+  "advancement.evilcraft.player_devastator": "毁灭玩家",
+  "advancement.evilcraft.player_devastator.desc": "用死灵法师权杖召唤僵尸杀死玩家。",
+  "advancement.evilcraft.power_crafting": "力量制造",
+  "advancement.evilcraft.power_crafting.desc": "制造充能的末影合成包!",
+  "advancement.evilcraft.cannibal": "食人族",
+  "advancement.evilcraft.cannibal.desc": "试试吃点人肉。",
+
+
+  "gui.exalted_crafting": "便携合成",
+  "gui.exalted_crafting.clear": "清除",
+  "gui.exalted_crafting.balance": "平均分配",
+  "gui.exalted_crafting.toggle_return": "切换返回模式",
+  "gui.waila.tank_config": "液体含量",
+  "gui.waila.inner_blocks_config": "容器内物品",
+  "gui.waila.box_of_eternal_closure_config": "永恒的灵魂箱所含灵魂",
+  "gui.waila.block_info_config": "按住Shift查看方块详情",
+  "gui.waila.broom_info_config": "扫帚信息",
+  "gui.nei.fluids": "液体",
+
+
+  "broom.evilcraft.shiftinfo": "<按下Shift以查看扫帚信息>",
+  "broom.parts.evilcraft.shiftinfo": "<按下Shift以查看扫帚部件信息>",
+  "broom.parts.evilcraft.rod_bare": "裸杆",
+  "broom.parts.evilcraft.rod_wood": "木杆",
+  "broom.parts.evilcraft.rod_stone": "石杆",
+  "broom.parts.evilcraft.rod_bone": "骨杆",
+  "broom.parts.evilcraft.rod_blaze": "烈焰棒杆",
+  "broom.parts.evilcraft.rod_reed": "甘蔗杆",
+  "broom.parts.evilcraft.rod_netherrack": "下届岩杆",
+  "broom.parts.evilcraft.rod_obsidian": "黑曜石杆",
+  "broom.parts.evilcraft.rod_undead": "亡灵木杆",
+  "broom.parts.evilcraft.rod_prismarine": "海晶石杆",
+  "broom.parts.evilcraft.rod_ice": "冰杆",
+  "broom.parts.evilcraft.rod_sponge": "海绵杆",
+  "broom.parts.evilcraft.rod_endstone": "末地石杆",
+  "broom.parts.evilcraft.rod_purpur": "紫珀杆",
+  "broom.parts.evilcraft.rod_bamboo": "竹竿",
+  "broom.parts.evilcraft.brush_bare": "裸刷",
+  "broom.parts.evilcraft.brush_wheat": "干草刷",
+  "broom.parts.evilcraft.brush_wool": "羊毛刷",
+  "broom.parts.evilcraft.brush_feather": "羽毛刷",
+  "broom.parts.evilcraft.brush_twig": "枝条刷",
+  "broom.parts.evilcraft.brush_leaves": "树叶刷",
+  "broom.parts.evilcraft.brush_honey": "蜂蜜刷",
+  "broom.parts.evilcraft.cap_bare": "裸端",
+  "broom.parts.evilcraft.cap_gem_dark": "暗黑宝石端",
+  "broom.parts.evilcraft.cap_gem_dark_power": "充能暗黑宝石端",
+  "broom.parts.evilcraft.cap_gem_diamond": "钻石端",
+  "broom.parts.evilcraft.cap_gem_emerald": "绿宝石端",
+  "broom.parts.evilcraft.cap_gem_quartz": "石英端",
+  "broom.parts.evilcraft.cap_gem_lapis": "青金石端",
+  "broom.parts.evilcraft.cap_head_skeleton": "骷髅头颅端",
+  "broom.parts.evilcraft.cap_head_witherskeleton": "凋零骷髅头颅端",
+  "broom.parts.evilcraft.cap_head_zombie": "僵尸头颅端",
+  "broom.parts.evilcraft.cap_head_player": "玩家头颅端",
+  "broom.parts.evilcraft.cap_head_creeper": "爬行者头颅端",
+  "broom.parts.evilcraft.cap_head_wither": "凋零头颅端",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "broom.parts.evilcraft.cap_head_wither": "凋灵头颅端",

```